doc/sphinx/language/gallina-extensions.rst

```diff
@@ -2189,12 +2189,6 @@ an inductive type or any constant with a type of the form
 Then the user is able to apply an object that is not a function, but
 can be coerced to a function, and more generally to consider that a
 term of type ``A`` is of type ``B`` provided that there is a declared coercion
-between ``A`` and ``B``. The main command is
-
-.. cmd:: Coercion @qualid : @class >-> @class.
-
-which declares the construction denoted by qualid as a coercion
-between the two given classes.
```

Comment:
What is that change?

Reply:
This command had two entries in the table, in two chapters. It is not supported by our new infrastructure, but anyway this entry was mainly pointing to the other one, so I simply removed it.

Reply:
You removed too much (or not enough).

Reply:
Ouch! I guess I haven't slept enough. Good catch :)